src/main/webapp/placeGuides/ListPlaceGuideDisplayer.js

```diff
@@ -2,75 +2,132 @@
  * This class is responsible for representing
  * the placeGuides on a scrollable list.
  */
-class ListPlaceGuideDisplayer {
-  constructor() {
-    this._placeGuidesOnList = {};
-  }
 
-  update(placeGuides) {
-    this.removePreviousPlaceGuidesFromList(placeGuides);
-    this.addNewPlaceGuidesToList(placeGuides);
-  }
+class ListPlaceGuideDisplayer {
 
-  removeFromList(placeGuideId) {
+  static QUERY = {
+    "ALL_PUBLIC_IN_MAP_AREA": {
+      listTitle: "Discover Guides",
+      listSubTitle: "in selected map area"
+    },
+    "CREATED_ALL_IN_MAP_AREA": {
+      listTitle: "My Guides",
+      listSubTitle: "in selected map area"
+    },
+    "BOOKMARKED": {
+      listTitle: "Bookmarked Guides",
+      listSubTitle: ""    }
+  };
 
+  constructor(placeGuideDisplayQuery) {    
+    this._listPlaceGuideDisplayerDiv = document.getElementById("listPlaceGuideDisplayer");
+    this._listPlaceGuideDisplayerDiv.classList.add(
+        "list-group", "my-place-guide-list", "list-place-guide-displayer", "form-card");
+    this._placeGuideDisplayQuery = placeGuideDisplayQuery;
+    this._listPlaceGuideDisplayerDiv.appendChild(
+        this.createListTitle(
+            ListPlaceGuideDisplayer.QUERY[placeGuideDisplayQuery]));
   }
 
-  addToList(placeGuideId) {
+  get listPlaceGuideDisplayerDiv() {
+    return this._listPlaceGuideDisplayerDiv;
+  }
 
+  update(placeGuides) {
+    this.removeAllPlaceGuidesFromList();
+    this._listPlaceGuideDisplayerDiv.appendChild(
+        this.createListTitle(
+            ListPlaceGuideDisplayer.QUERY[this._placeGuideDisplayQuery]));
```

Comment:
Don't remove the header each time we update the placeguides.

Reply:
modified the update method to note remove the header on updating the place guides.